src/main/java/frc/robot/Robot.java

```diff
@@ -195,8 +197,14 @@ public void teleopInit() {
 
     mStorage.init();
     mDrive.init();
-
-    Config.getInstance().reload();
+    
+    // updated in Intake.java
```

Comment:
Why are these in both Robot.java and Intake.java?

Reply:
They are initially set in Robot.java but updated in Intake